src/Statistics/MetricsPerMessageType.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Statistics;
+
+/**
+ * @internal
+ */
+final class MetricsPerMessageType
+{
+    private $from;
+    private $to;
```

Comment:
Maybe `$fromDate` and `$toDate` - same in `Statistics`. It's not really about defining their php type in their name (after all, I wouldn't want a property called `$firstNameString`) - it's about making it clear that these define a date range - as opposed to a number range (count from 1 to 100!).